satellite/satellitedb/usercredits.go

```diff
@@ -98,17 +99,17 @@ func (c *usercredits) UpdateAvailableCredits(ctx context.Context, creditsToCharg
 			break
 		}
 
-		creditsForUpdate := credit.CreditsEarnedInCents - credit.CreditsUsedInCents
+		creditsForUpdateInCents := credit.CreditsEarnedInCents - credit.CreditsUsedInCents
```

Comment:
Think I've asked this before but can this value be negative ?

Reply:
I do not know the answer to this question. @fuskovic / @VinozzZ?

Reply:
No, this value shouldn't be negative here. On line 82, we are only getting credits that has `CreditsEarnedInCents > CreditsUsedInCents`